android/app/src/main/res/layout/activity_speakers.xml

```diff
@@ -17,7 +17,7 @@
         <android.support.design.widget.CollapsingToolbarLayout
             android:id="@+id/collapsing_toolbar"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
+            android:layout_height="@dimen/item_image_view_extra_large"
```

Comment:
In the addition of ViewModel for the Activity, i don't understand why you need to change the layout for. Please keep the PR focused on a single issue. If you want to make changes, then create an issue for it and get it approved before working.

Reply:
Actually i didn't make this particular change.

Reply:
Pardon, i didn't get you . If you didn't make these changes how are they here ?

Reply:
I am sorry i don't get this. I never touched the layout file. And now it chows even more changes to it in the last commit. How should i revert them?

Reply:
They crept in maybe because i accidentally merged wrong branches while trying to update my fork. I have corrected them.